### DISCUSSION TITLE
11월 21일 주간회의 쓰레드

### DISCUSSION THREAD

**Discussion by wonsss** [woowacourse/prolog#1119] [General]
11월 21일 주간회의 쓰레드

**Comment by wonsss** [comment]
### 로드맵 어드민에서 키워드 관리 페이지 구현 완료
https://github.com/woowacourse/prolog-admin/pull/17
- [x] 세션 선택 > 최상위 키워드 선택 > 최대 3뎁스 키워드까지 선택 가능
- [x] 선택된 키워드에서 키워드의 콘텐츠를 수정, 삭제할 수 있고 키워드 내에 새 키워드를 생성할 수 있음

![로드맵 키워드](https://user-images.githubusercontent.com/59413128/202942681-07d84310-d0b4-4a89-b83f-23dbdf9720d9.gif)


### 로드맵 어드민 관련 추가 구현해야 할 기능
- 퀴즈 관리 기능 등

with 시지프(@euijinkk)

**Reply by her0807** [reply]
와 마르코 빠르다

**Comment by wishoon** [comment]
### 로드맵 (Keyword, Quiz) 목 API 구현 완료
- https://github.com/woowacourse/prolog/pull/1114 (이브)

### 로드맵 (Keyword) API 리뷰 완료 후 PR 머지
- https://github.com/woowacourse/prolog/pull/1098 (루키)

### 로드맵 (Quiz) API 코드 리뷰 중 (오늘 중으로 머지)
- https://github.com/woowacourse/prolog/pull/1088 (수달)

### 남은 일정
- feature/1056-roadmap 브랜치를 develop 브랜치로 머지

with 수달(@her0807), 이브(@2yujeong)

**Comment by EastHShin** [comment]
### ERD 문서 및 인가과정 문서 Wiki로 이관
[인가 과정](https://github.com/woowacourse/prolog/wiki/%EC%9D%B8%EA%B0%80-%EA%B3%BC%EC%A0%95)
[ERD 한눈에 보기](https://github.com/woowacourse/prolog/wiki/ERD-%ED%95%9C%EB%88%88%EC%97%90-%EB%B3%B4%EA%B8%B0)
- ERD 사진 밑에 테이블에 대한 간략한 설명 문서 추가
